_episodes/12.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: episode
+title:  "12: Trino gets super visual with Superset!"
```

Comment:
Apache Superset

Maybe

Reply:
I am more in favor of keeping the title short...and we specify Apache Superset in the notes frequently.